ui/helpers/constants.js

```diff
@@ -302,8 +298,25 @@ export const phAlertStatusMap = {
   CONFIRMING: { id: 5, text: 'confirming' },
 };
 
-export const phCompareBaseLineDefaultTimeRange = 86400 * 2;
+export const compareDefaultTimeRange = 86400 * 2;
 
 export const thBugSuggestionLimit = 20;
 
 export const thMaxPushFetchSize = 100;
+
+// we only want bug_ui and tree_ui to be used for UI validation
+// (for Intermittent Failures View) because if there is a valid
+// type used but its a non-existent repo or bug_id
+// we want to see that message from the api response
+export const prettyErrorMessages = {
+  startday: 'startday is required and must be in YYYY-MM-DD format.',
+  endday: 'endday is required and must be in YYYY-MM-DD format.',
+  bug_ui: 'bug is required and must be a valid integer.',
+  tree_ui:
+    'tree is required and must be a valid repository or repository group.',
+  default: 'Something went wrong.',
+  status503:
+    'There was a problem retrieving the data. Please try again in a minute.',
```

Comment:
I think we should inline most of these, since in many cases they will only be used once - or if they are currently used more than once, the best place to de-duplicate will be at a higher level (eg the wrapper around `fetch()`).

Reply:
Some of these, like `startday` and `endday` are used more than once - for UI validation and for server side error handling in the `processErrorMessage` helper below, but only for IFV. So do we want to have one object that's shared that contains error messages for different apps, or each app with its own specific error messages object? Perfherder seems to have quite a few query params that will need validation so the question is whether they should live here or in their own file.

Also, a relevant question is whether we want meaningful messages only for UI validation but just deliver whatever is returned for server side responses.